UseCase50/README.md

```diff
@@ -0,0 +1,40 @@
+# Defines the fitness of Kōjō for
+- Basic functionality in the 5.x branch
+
+## Setup UseCase50
+
+### Running Kōjō
+Setting up the containers using docker-compose:
+
+```bash
+# Get the latest repo
+git clone git@github.com:neighborhoods/KojoFitness.git;
+cd KojoFitness;
+git checkout 4.x;
```

Comment:
```suggestion
git checkout 5.x;
```

Reply:
It should actually be `master` like we discussed offline, I've pushed the change